package.json

```diff
@@ -41,6 +41,7 @@
     "globby": "^13.1.1",
     "minimist": "^1.2.5",
     "node-fetch": "^3.2.3",
+    "prompts": "^2.4.2",
```

Comment:
Prompts comes with types?

Reply:
Yes, it comes with types. I am not sure about typescript but it also has this: https://www.npmjs.com/package/@types/prompts

Reply:
![image](https://user-images.githubusercontent.com/52219366/162585263-c542b4e4-5430-4b65-b42c-076568530379.png)

yes, it comes with types. Auto complete working fine on vscode as well.

Reply:
So no?

Reply:
> So no?

It comes with types.